install.js

```diff
@@ -417,6 +417,9 @@ function configureMasterTenant(callback) {
   installHelpers.showSpinner('Starting server');
   // run the app
   app.run({ skipVersionCheck: true });
+  app.on('serverStartError', function(error) {
+    return exit(1, error.message || error);
+  });
```

Comment:
It looks like this just replicates the error handling in `lib/application.js`

Reply:
It also logs the error message that connection failed.
Without that I don't get this message.